pkgs/applications/office/hledger-check-fancyassertions/default.nix

```diff
@@ -0,0 +1,40 @@
+{lib, stdenv, haskellPackages, fetchurl, writers, ghc}:
+
+stdenv.mkDerivation rec {
+  pname = "hledger-check-fancyassertions";
+  version = haskellPackages.hledger-lib.version;
+
+  src = fetchurl {
+    url = "https://raw.githubusercontent.com/simonmichael/hledger/hledger-lib-${version}/bin/hledger-check-fancyassertions.hs";
```

Comment:
I don't think this is a good idea. I would prefer a stable url, which contains the commit id. Because of the fixed hash, this solution will break with the next hledger release but we won't notice because hydra will still have the output with this hash.

Reply:
I understand what you say and changed my code but I don't understand why you would want me to use a commit id instead of a tag.

Reply:
Yeah, you are completely right. I hadn‘t considered this solution, it is certainly better.